n2y/export.py

```diff
@@ -60,7 +66,9 @@ def export_page(
 ):
     pandoc_ast = page.to_pandoc()
 
-    if (number_empty_headers := _count_headerless_tables(pandoc_ast)) > 0:
+    if (number_empty_headers := _count_headerless_tables(pandoc_ast)) > 0 and (
```

Comment:
This line is too complicated. Split it out into smaller lines. E.g., move `number_empty_headers := _count_headerless_tables(pandoc_ast)` onto its own line.